### PR TITLE
Add typescript alias into the project for simpler imports

### PR DESCRIPTION
**Pull request by makivlach**
There are a couple of relative imports in the project, making navigating imports a little scary. If you change the project structure and move something to a different directory, you must adjust all the imports in the project. Using a typescript path alias makes this situation much more comfortable since it can be edited from one place.

### CODE REVIEW

File: src/models/EET.ts [hunk 1]
```diff
@@ -6,13 +6,14 @@
...
  */
 
-import axios from "axios";
-import { GoPay } from "..";
-import { eet } from "../types/eet";
+import { GoPay } from "~/goPay";
+import { eet } from "~/types/eet";
 
 export class Misc {
+  // @ts-ignore
```
Comment (OceanTechnic) [inline src/models/EET.ts:13]: Do we need the `// @ts-ignore`?

File: src/models/EET.ts [hunk 2]
```diff
@@ -6,13 +6,14 @@
...
   private __client: GoPay;
-    private __sufix = "/payments/payment";
+  // @ts-ignore
+  private __sufix = "/payments/payment";
```
Comment (OceanTechnic) [inline src/models/EET.ts:16]: Isn't this a unsed variable?

**Review by OceanTechnic — CHANGES_REQUESTED** [review]
_I would stop the tracking of lock files_ **edit: my bad**
And I had 2 comments 

Othervise LGTM

File: src/models/EET.ts (hunk 1 above)
Comment (makivlach) [reply src/models/EET.ts:13]: The IDE was telling me that these properties weren’t used anywhere and were throwing me red underline, which was driving me crazy 😁

File: src/models/EET.ts (hunk 2 above)
Comment (makivlach) [reply src/models/EET.ts:16]: Yep, it is. I would delete it too, but I don't know why it got into the master in the first place. I guess it was for some reason. That is why I left it there.

**Comment by makivlach** [comment]
> I would stop the tracking of lock files
> 
> And I had 2 comments 
> 
> 
> 
> Othervise LGTM

Why would you stop tracking lock files? It is common practice to have them/it included in the repo to let the package manager know what package versions it should install. 

I would leave it, because if some newer version of some package breaks for some reason (e.g. compatibility issue with a newer node or some other package) the lock file would let everybody forking this package know: “these package versions are stable together”.  Otherwise, without .lock files, it could install the newest package versions when installing on the new machine - making the behaviour of the app inconsistent with colleagues.

These situations, unfortunately, happen often, trust me.

File: src/models/EET.ts (hunk 1 above)
Comment (OceanTechnic) [reply src/models/EET.ts:13]: Ah okie then 👍🏻

File: src/models/EET.ts (hunk 2 above)
Comment (OceanTechnic) [reply src/models/EET.ts:16]: @0ndras3k Can you take look on the variable?

**Comment by OceanTechnic** [comment]
Hmm, okay then my bad. 
I haven't said anything 😄

**Pull request merged**